hoomd/logging.py

```diff
@@ -657,14 +625,20 @@ class they come from. For instance, the `hoomd.md.pair.LJ` class has a
         occur for user specified names that are reused.
 
     Args:
-        categories (`list` of `str`, optional): A list of string categories
+        categories (`list` [`str`], optional): A list of string categories
             (list of categories can be found in `LoggerCategories`). These are
             the only types of loggable quantities that can be logged by this
             logger. Defaults to allowing every type.
         only_default (`bool`, optional): Whether to log only quantities that are
             logged by default. Defaults to ``True``. Non-default quantities
             are typically measures of operation performance rather than
             information about simulation state.
+
+    .. rubric:: Example:
+
+    .. code-block:: python
+
+        logger = hoomd.logging.Logger()
```

Comment:
See my PR #1588 where I added example logger constructions. We should choose one to keep.

Reply:
If you think the additional examples are helpful, we can keep them. I have been avoiding code comments in examples. If additional explanation or a header is needed, put that outside the code block. For example: https://github.com/glotzerlab/hoomd-blue/blob/4e30e11957421a349da1371a6980c12497d058cf/hoomd/md/methods/methods.py#L107-L123